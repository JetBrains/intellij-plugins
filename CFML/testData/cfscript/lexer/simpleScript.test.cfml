
<cfscript>
//a++;
var a = 2;
</cfscript>