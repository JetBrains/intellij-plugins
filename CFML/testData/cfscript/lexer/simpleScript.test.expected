WHITE_SPACE ('
')
OPENER ('<')
CF_TAG_NAME ('cfscript')
R_ANGLEBRACKET ('>')
WHITE_SPACE ('
')
COMMENT ('//a++;')
WHITE_SPACE ('
')
VAR_KEYWORD ('var')
WHITE_SPACE (' ')
IDENTIFIER ('a')
WHITE_SPACE (' ')
ASSIGN ('=')
WHITE_SPACE (' ')
INTEGER ('2')
SEMICOLON (';')
WHITE_SPACE ('
')
LSLASH_ANGLEBRACKET ('</')
CF_TAG_NAME ('cfscript')
CLOSER ('>')